docs/getting-started/verify.md

```diff
@@ -16,9 +16,8 @@ Verifying the container image is pretty straightforward:
 
 ```bash
 $ RELEASE='v0.23.0'
-$ URL="https://github.com/inspektor-gadget/inspektor-gadget/blob/main/inspektor-gadget.pub"
 # We first need to get the public key.
-$ wget $URL
+$ wget "https://raw.githubusercontent.com/inspektor-gadget/inspektor-gadget/${RELEASE}/inspektor-gadget.pub"
 $ cosign verify --key inspektor-gadget.pub ghcr.io/inspektor-gadget/inspektor-gadget:${RELEASE}
```

Comment:
Why not directly do the following as I suggested [here](https://github.com/inspektor-gadget/inspektor-gadget/pull/2678/files#r1561276377)?

```suggestion
cosign verify --key https://raw.githubusercontent.com/inspektor-gadget/inspektor-gadget/${RELEASE}/inspektor-gadget.pub ghcr.io/inspektor-gadget/inspektor-gadget:${RELEASE}
```

Reply:
I didn't know it was possible. Updated it!